src/libs/API/parameters/SendInvoiceParams.ts

```diff
@@ -0,0 +1,19 @@
+type SendInvoiceParams = {
+    senderWorkspaceID: string;
+    accountID: number;
+    receiverEmail?: string;
```

Comment:
Nice little pro tip: _either_ `receiverEmail` _or_ `receiverInvoiceRoomID` is required. So I see why you made them both optional, but it later led to a problem [here](https://github.com/Expensify/App/issues/43060). So what you can do is use [RequireAtLeastOne](https://github.com/sindresorhus/type-fest/blob/main/source/require-at-least-one.d.ts) from type-fest.